gluoncv/data/transforms/presets/ssd.py

```diff
@@ -211,3 +221,128 @@ def __call__(self, src, label):
         img = mx.nd.image.to_tensor(img)
         img = mx.nd.image.normalize(img, mean=self._mean, std=self._std)
         return img, bbox.astype(img.dtype)
+
+class SSDCocoDALIPipeline(Pipeline):
+    """DALI Pipeline with COCO Reader and SSD training transorm.
```

Comment:
```suggestion
    """DALI Pipeline with COCO Reader and SSD training transform.
```
Waiting for further review before committing the change